gsy_framework/sim_results/electric_blue/aggregate_results.py

```diff
@@ -0,0 +1,145 @@
+from collections import defaultdict
+from dataclasses import dataclass, field, fields
+from typing import Dict, List
+
+from pendulum import DateTime
+
+from gsy_framework.enums import AggregationResolution, AvailableMarketTypes
+
+# Used by forward markets; the following dictionary defines
+# what aggregations are needed for each market type.
+MARKET_RESOLUTIONS = {
+    AvailableMarketTypes.YEAR_FORWARD: [AggregationResolution.RES_1_MONTH],
```

Comment:
I probably missed a discussion about this, but why do you use enums here and not directly assign a duration here?
```suggestion
    AvailableMarketTypes.YEAR_FORWARD: duration(months=1),
```

Reply:
We couldn't easily persist `Duration` objects in DB, so we created this enum.

Reply:
Ah, ok. But I am wondering: Will we save the timeseries data returned by `ForwardDeviceTimeSeries` in the DB for different resolutions? I see that you wrote this in point 8 in the PR description. But I fail to find the point where this is actually done. Can you please help me? Thanks!

Reply:
Yes. The `generate` method on `ForwardDeviceTimeSeries` accepts the resolution to calculate. The returned data points will then be saved to DB. However, I'm not sure whether we save each each data point as a DB row or we store one giant json containing all data point. The saving operation will be done by @cruky in the next step.

Reply:
I see, thanks!
This is a discussion about performance I think. If we want to make high resolution trade profiles available very quickly, we probably need to pre-calculate these and store them into the DB as soon as there are new markets slot started.
If the delay due to the calculation is not so large, I would vote to not clutter the DB. 
Tagging @spyrostz that has probably spent some thought on this

Reply:
I think that it is better to save them in the database because as Hannes mentioned, we need to make high resolution profiles available very quickly.  Have a look at this selector on the energy bought / sold graph: 

<img width="438" alt="Screenshot 2022-09-14 at 10 49 39" src="https://user-images.githubusercontent.com/37290802/190093981-e7dd7cea-e008-4097-a91e-58f98656cb45.png">

In order to do this reasonably fast, as our experience with d3a.io showed, it needs to be precalculated.